### PR TITLE
changed from zesty to xenial

### LINKED ISSUE #235: Zesty releases are not found
- Issue type: Bug report

### Short description

when checking travis logs of pull #234 
the error is 
> Failed getting release file http://archive.ubuntu.com/ubuntu/dists/zesty/Release

### Environment
Travis CI 
### Steps to reproduce
try using build script from pull #234 

Zesty release is not found in http://archive.ubuntu.com/ubuntu/dists/

### solution 
switch to a different build

### PR DESCRIPTION
**Pull request by xeon-zolt**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message Fixes #235

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -16,7 +16,8 @@ mirror=${2:-"http://archive.ubuntu.com/ubuntu/"}
 # Install language with the most popcon
 gnomelanguage=${3:-'{en}'}	
 # Release name, used by debootstrap.  Examples: lucid, maverick, natty.
-release=${4:-zesty}
+#release=${4:-zesty}
```
Comment (abishekvashok) [inline build.sh]: Can't we remove it?

Comment (xeon-zolt) [reply build.sh]: we can
but the build is still failing

Comment (abishekvashok) [reply build.sh]: Keep in mind these are not only the files to change. You are getting your sources from the sources file.

**Comment by tabesin** [comment]
Yes, the sources are still missing

**Pull request merged**